### DISCUSSION TITLE
Auto detect browser language

### DISCUSSION THREAD

**Discussion by WoLfulus** [directus/directus#3650] [Feature Requests]
When not logged in, the login page renders as english by default, it would be nice to have it using the user's browser language in order to infer the user's preferred language.

**Comment by rbraggaar** [comment]
An option to set a different default in the project settings would also be very helpful.

**Reply by azrikahar** [reply]
#8196 should fit this use case 👍

**Reply by rbraggaar** [reply]
That is great to hear. Thankful to all contributors again.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
